integration/kubernetes/helm-generate.sh

```diff
@@ -22,7 +22,10 @@ function printUsage {
   echo
   echo "UFS is only for single-ufs mode. It should be one of:"
   echo -e " local             \t Use a local destination for UFS journal."
-  echo -e " hdfs              \t Use HDFS for UFS journal"
+  echo -e " hdfs              \t Use HDFS for UFS journal."
+  echo
+  echo "OPTS allows parameters. Currently the options are:"
+  echo -e " --enable-workerFuse\tEnable FUSE in worker container."
```

Comment:
--worker-fuse Launch FUSE application in the worker process inside worker container.